### PR TITLE
Pushing test programs to the master branch.

### LINKED ISSUE #9: Create Test "user programs"
- programs we can use to simulate user defined programs and test our software with.

##TODO

### PR DESCRIPTION
**Pull request by iliescua**
# Date

Initial PR Date: 12/7/2020

Due Date: 12/7/2020

# Reviewers

@rockboynton 

# Description

Created two C++ programs that are used to blink the USER0 and USER1 LEDs on the BeagleBone at different frequencies. A shell script was also created that when run, will add folder called test_programs to the users home directory as well as compile the two C++ programs using g++ and the -o flag. In order to run the programs type the command ./<name of the program> and it will execute. You should the USER0 and USER1 flash at differing frequencies. 

# Project Issue

[Create Test "user programs" #9](https://github.com/Senior-Design-0x07/the-hobby-hub/issues/9)

# Test Results

The programs are verified because they compiled and the USER0 and USER1 LEDs flashed at varying frequencies as they were supposed to ensuring that everything was running correctly.

### CODE REVIEW

File: files/test_programs/light0.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using namespace std;
+
+#define LED0_PATH "/sys/class/leds/beaglebone:green:usr0"
+
+int main(int argc, char* argv[]){
+	fstream fs;
+	
+	while(1) {
+		fs.open(LED0_PATH "/brightness", fstream::out);
+		fs << "1";
```
Comment (rockboynton) [inline files/test_programs/light0.cpp:14]: This is really the way to set an LED? Very strange compared to the way we do it in embedded

File: scripts/installTests.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+mkdir -p /etc/hobby-hub/
+cp -r files/test_programs /etc/hobby-hub/
+
+sudo g++ /etc/hobby-hub/test_programs/light0.cpp -o /etc/hobby-hub/test_programs/light0
+sudo g++ /etc/hobby-hub/test_programs/light1.cpp -o /etc/hobby-hub/test_programs/light1
```
Comment (rockboynton) [inline scripts/installTests.sh]: Add newline at end of the file
```suggestion
sudo g++ /etc/hobby-hub/test_programs/light1.cpp -o /etc/hobby-hub/test_programs/light1

```

File: files/test_programs/light0.cpp (hunk 1 above)
Comment (iliescua) [reply files/test_programs/light0.cpp:14]: This is only the case for the USER LEDs on the board. I do not have a breadboard or LEDs to use the GPIO pins to configure them.

**Review by rockboynton — COMMENTED** [review]
Looks good

**Pull request merged**